src/main/java/org/isf/medicalinventory/gui/InventoryWardEdit.java

```diff
@@ -0,0 +1,505 @@
+        return cancelButton;
+    }
+
+    private JButton getPrintButton() {
+        printButton = new JButton(MessageBundle.getMessage("angal.common.print.btn"));
+        printButton.setMnemonic(MessageBundle.getMnemonic("angal.common.print.btn.key"));
+        return printButton;
+    }
+
+    private JButton getCloseButton() {
+        closeButton = new JButton(MessageBundle.getMessage("angal.common.close.btn"));
+        closeButton.setMnemonic(MessageBundle.getMnemonic("angal.common.close.btn.key"));
+        closeButton.addActionListener(new ActionListener() {
+            public void actionPerformed(ActionEvent e) {
+                dispose();
+            }
+
+        });
+        return closeButton;
+    }
+
+    private JScrollPane getScrollPaneInventory() {
+        if (scrollPaneInventory == null) {
+            scrollPaneInventory = new JScrollPane();
+            scrollPaneInventory.setViewportView(getJTableInventoryRow());
+        }
+        return scrollPaneInventory;
+    }
+
+    private JTable getJTableInventoryRow() {
+        if (jTableInventoryRow == null) {
+            jTableInventoryRow = new JTable();
+            jTetFieldEditor = new JTextField();
+            jTableInventoryRow.setFillsViewportHeight(true);
+            jTableInventoryRow.setModel(new InventoryRowModel());
+            jTableInventoryRow.getSelectionModel().addListSelectionListener(new ListSelectionListener() {
+
+                @Override
+                public void valueChanged(ListSelectionEvent e) {
+
+                    if (!e.getValueIsAdjusting()) {
+                        jTableInventoryRow.editCellAt(jTableInventoryRow.getSelectedRow(), 3);
+                        jTetFieldEditor.selectAll();
+                    }
+
+                }
+            });
+            DefaultCellEditor cellEditor = new DefaultCellEditor(jTetFieldEditor);
+            jTableInventoryRow.setDefaultEditor(Double.class, cellEditor);
+        }
+        return jTableInventoryRow;
+    }
+
+    class InventoryRowModel extends DefaultTableModel {
+
+        private static final long serialVersionUID = 1L;
+
+        public Class<?> getColumnClass(int c) {
+            if (c == 0) {
+                return String.class;
+            } else if (c == 1) {
+                return String.class;
+            } else if (c == 2) {
+                return Double.class;
+            } else if (c == 3) {
+                return Double.class;
+            }
+            return null;
+        }
+
+        public int getRowCount() {
+            return 0;
+        }
+
+        public String getColumnName(int c) {
+            return pColums[c];
+        }
+
+        public int getColumnCount() {
+            return pColums.length;
+        }
+
+        @Override
+        public boolean isCellEditable(int rowIndex, int columnIndex) {
+            return columnEditable[columnIndex];
+        }
+    }
+
+    public String formatDateTime(GregorianCalendar time) {
```

Comment:
Where is this method used?   And I believe 'GregorianCalendar` is no longer used in the project.